2-Browsers/Week1/homework/ex4-whatsTheTime/index.html

```diff
@@ -6,6 +6,7 @@
     <title>What's The Time?</title>
   </head>
   <body>
+    <!-- Reference: https://codepen.io/afarrar/pen/JRaEjP -->
```

Comment:
It looks like you copied the solution to this exercise instead of creating it yourself.
While it's good to look online for inspiration and or solutions to parts of the exercise I think it's more beneficial for your learning process if you solve the exercises yourself!

Reply:
I changed the code. BTW, I will do exercises myself.😊